test/spec.js

```diff
@@ -1092,6 +1092,15 @@ module.exports = (fetchMock, theGlobal, Request, Response) => {
 					})
 			});
 
+			it('takes an optional Promise to use', () => {
+				const BluebirdPromise = require('bluebird');
+				const sbx = fetchMock
+					.sandbox(BluebirdPromise)
+					.mock('http://example.com', 200);
+
+				expect(sbx('http://example.com')).to.be.instanceof(BluebirdPromise);
+			});
```

Comment:
Could you add an additional test(s0 to make sure sandboxed instances don't mess up promise references for other instances or the global fetch-mock instance